lib/browser/guest-view-manager.ts

```diff
@@ -223,7 +237,8 @@ const attachGuest = function (event, embedderFrameId, elementInstanceId, guestIn
   ]);
 
   // Inherit certain option values from embedder
-  const lastWebPreferences = embedder.getLastWebPreferences();
+  // Typing satisfy's no index signature error on lastWebPreferences in for loop
+  const lastWebPreferences: { [key: string]: any } = embedder.getLastWebPreferences();
```

Comment:
```suggestion
  const lastWebPreferences: Electron.WebPreferences = embedder.getLastWebPreferences();
```

Reply:
See comment on line 240 for why I could not do this.

Ran into this error for each associated variable in the loop:

```javascript
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'WebPreferences'.
  No index signature with a parameter of type 'string' was found on type 'WebPreferences'.
```

Googled around for a bit but couldn't find something that worked for this case.